src/components/Records/Search/Output/SearchOutput.vue

```diff
@@ -137,8 +133,3 @@ export default {
 }
 </style>
 
-<style scoped lang="scss">
```

Comment:
Thanks. I am not sure whether leaving lang="scss" causes more compile-time or not but in case of I'd say let's deal from now on not using it unless we are actually using sass? I will commit changes later.